deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -1414,6 +1419,14 @@ def reset_hosts(request, name, stage):
     return redirect('/env/{}/{}/hosts/'.format(name, stage))
 
 
+def reset_deploys_on_hosts(request, name, stage):
```

Comment:
we might not want to restart all deployments on all hosts in a stage. Restart all deployments on a host should be shown in the host view only.

Reply:
Changed